packages/snaps-controllers/src/test-utils/controller.ts

```diff
@@ -212,12 +264,17 @@ export const getPersistedSnapsState = (
 };
 
 // Mock controller messenger for Cronjob Controller
-export const getRootCronjobControllerMessenger = () =>
-  new ControllerMessenger<
+export const getRootCronjobControllerMessenger = () => {
+  const messenger = new MockControllerMessenger<
     CronjobControllerActions | AllowedActions,
     CronjobControllerEvents | AllowedEvents
   >();
 
+  jest.spyOn(messenger, 'call');
```

Comment:
What is this for?

Reply:
That's needed for `expect(messenger.call).toHaveBeenCalledWith(...)` to work.